controllers/imageclusterinstall_controller.go

```diff
@@ -764,15 +765,37 @@ func (r *ImageClusterInstallReconciler) writeInputData(
 			return fmt.Errorf("failed to generate extra manifests: %w", err)
 		}
 
+		configFilePath := clusterConfigPath
+		kubeconfig, kubeadmPassword, seedReconfig, secretsExist, err := r.Credentials.ClusterIdentitySecrets(ctx, cd)
```

Comment:
maybe better to use struct to store reinstall data like kubeconfig, kubeadmPassword, seedReconfig?

Reply:
:shrug: I could, but it would need to be returned here by the `credentials` package and then taken as input in `installer`

We don't currently have a dependency there, but I guess it wouldn't cause too much of an issue?